main/control/user.py

```diff
@@ -24,7 +24,8 @@
 @app.route('/admin/user/')
 @auth.admin_required
 def user_list():
-  user_dbs, user_cursor = model.User.get_dbs(email=util.param('email'))
+  user_dbs, user_cursor, prev_cursor =\
+      model.User.get_dbs(email=util.param('email'), prev_cursor=True)
```

Comment:
But on the other hand.. there will never be multiple previous cursors as you can simply ignore by not setting the `prev_cursor=True`, but you can't skip the `next_cursor` from the output.. (at least as it's implemented at the moment)